### PR TITLE
Url pointing to install.yaml pointing to master

### PR DESCRIPTION
**Pull request by mathieuduperre**
url was pointing to 0.7.0 install.yaml which was not the latest.

### CODE REVIEW

File: site/content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ We can install Agones to the cluster using the
 
 ```bash
 kubectl create namespace agones-system
-kubectl apply -f https://github.com/GoogleCloudPlatform/agones/raw/release-0.7.0/install/yaml/install.yaml
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/master/install/yaml/install.yaml
```
Comment (markmandel) [inline site/content/en/docs/Installation/_index.md]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/{{< release-branch >}}/install/yaml/install.yaml
```

Nice pickup!

Looks like we missed a spot when doing the dynamic update (we actually don't want to point at master, as master is development,  and development could always be broken)

**Comment by markmandel** [comment]
Gentle bump on this - if you aren't going to be able to complete this @mathieuduperre , do you mind if someone takes this PR over?

File: site/content/en/docs/Installation/_index.md (hunk 1 above)
Comment (markmandel) [reply site/content/en/docs/Installation/_index.md]: Not sure if you meant to include the suggested change in the PR? But we need the change to accept it.

**Comment by mathieuduperre** [comment]
thx to GDC, multitasking is not working well.

**Comment by markmandel** [comment]
Hah! Tell me about it.

Can you also squash this into a single commit, and I'll happily approve this 👍

**Comment by markmandel** [comment]
Gently bumping this once more - if you don't have time for this, totally cool. Happy to take it over - just want to have this correct in the documentation :+1:

**Comment by mathieuduperre** [comment]
feel free to take over, I don't have the project on my laptop to squash it and it may be a while until I find times to dig this. thanks and my apology.

**Comment by markmandel** [comment]
No worries at all! Life happens. I always like to ask permission before taking over PRs! Will jump on it.

**Comment by markmandel** [comment]
Closing and opening a new PR!

**Pull request closed without merging**